File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/system/MdiFormMainMenuItemInfo.java
package nta.med.data.model.ihis.system;

public class MdiFormMainMenuItemInfo {
	private String pgmSysId;
    private String pgmId;
	public MdiFormMainMenuItemInfo(String pgmSysId, String pgmId) {
		super();
		this.pgmSysId = pgmSysId;
		this.pgmId = pgmId;
	}
	public String getPgmSysId() {
		return pgmSysId;
	}
	public void setPgmSysId(String pgmSysId) {
		this.pgmSysId = pgmSysId;
	}
	public String getPgmId() {
		return pgmId;
	}
	public void setPgmId(String pgmId) {
		this.pgmId = pgmId;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/System/MdiFormMainMenuItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.System
{
    [Serializable]
	public class MdiFormMainMenuItemInfo
	{
		private String _pgmSysId;
		private String _pgmId;

		public String PgmSysId
		{
			get { return this._pgmSysId; }
			set { this._pgmSysId = value; }
		}

		public String PgmId
		{
			get { return this._pgmId; }
			set { this._pgmId = value; }
		}

		public MdiFormMainMenuItemInfo() { }

		public MdiFormMainMenuItemInfo(String pgmSysId, String pgmId)
		{
			this._pgmSysId = pgmSysId;
			this._pgmId = pgmId;
		}

	}
}